Description: Change /usr/dict/words to /usr/share/dict/words to reflect FHS.
Origin: vendor
Forwarded: not needed
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2011-11-26

--- a/lib/Data/Random/WordList.pm
+++ b/lib/Data/Random/WordList.pm
@@ -130,7 +130,7 @@
 
   use Data::Random::WordList;
 
-  my $wl = new Data::Random::WordList( wordlist => '/usr/dict/words' );
+  my $wl = new Data::Random::WordList( wordlist => '/usr/share/dict/words' );
 
   my @rand_words = $wl->get_words(10);
 
--- a/lib/Data/Random.pm
+++ b/lib/Data/Random.pm
@@ -580,7 +580,7 @@
 
 =item *
 
-wordlist - the path to the wordlist file.  A lot of systems have one at /usr/dict/words.  You can also optionally supply a Data::Random::WordList object to keep a persistent wordlist.  The default is the wordlist distributed with this module.
+wordlist - the path to the wordlist file.  On Debian systems one can be found at /usr/share/dict/words.  You can also optionally supply a Data::Random::WordList object to keep a persistent wordlist.  The default is the wordlist distributed with this module.
 
 =item *
 
